### ISSUE TITLE
Credit card create is deprecated

### ISSUE THREAD

**Issue by andrewblake1** [tuyakhov/yii2-braintree#3]
Thanks for the extension.

Just going thru integrating with my application now, noticed credit card create is deprecated so though I would mention.

**Comment by tuyakhov** [comment]
Yep, thanks for mentioning it. Unfortunately i don't have enough time to maintain this extension. I will try to fix it as soon as possible.
